diff --git src/ioc/dbtemplate/test/Makefile src/ioc/dbtemplate/test/Makefile
index 30e748983..f2f73daf4 100644
--- src/ioc/dbtemplate/test/Makefile
+++ src/ioc/dbtemplate/test/Makefile
@@ -11,7 +11,7 @@ include $(TOP)/configure/CONFIG
 TESTPROD_HOST_DEFAULT = dbltExpand
 TESTPROD_HOST_WIN32 = -nil-
 dbltExpand_SRCS += dbltExpand.c
-dbltExpand_LIBS += dbCore Com
+dbltExpand_LIBS += dbCore Com ca
 
 TESTS += msi
 
