diff --git src/tools/EpicsHostArch.pl src/tools/EpicsHostArch.pl
index e8e49bc5e..0314a8bfc 100644
--- src/tools/EpicsHostArch.pl
+++ src/tools/EpicsHostArch.pl
@@ -49,7 +49,12 @@ sub HostArch {
             }
             die "$0: macOS CPU type '$cpu' not recognized\n";
         }
-
+        if (m/^powerpc64-linux/) {
+            for ($cpu) {
+		return 'linux-ppc64e6500' if m/^ppc64/;
+            }
+            die "$0: linux-ppc64 OS CPU type '$cpu' not recognized\n";
+        }
         die "$0: Architecture '$arch' not recognized\n";
     }
 }
