diff --git modules/pva2pva/pdbApp/Makefile pdbApp/Makefile
index 37844d6..f55ba33 100644
--- modules/pva2pva/pdbApp/Makefile
+++ modules/pva2pva/pdbApp/Makefile
@@ -37,6 +37,7 @@ softIocPVA_SRCS += softMain.cpp
 softIocPVA_SRCS += softIocPVA_registerRecordDeviceDriver.cpp
 
 softIocPVA_LIBS += qsrv
+softIocPVA_LIBS += nt pvDatabase
 softIocPVA_LIBS += $(EPICS_BASE_PVA_CORE_LIBS)
 softIocPVA_LIBS += $(EPICS_BASE_IOC_LIBS)
 
