diff --git startup/EpicsHostArch.pl startup/EpicsHostArch.pl
index 09f7ffddd..b06c08b8a 100755
--- startup/EpicsHostArch.pl
+++ startup/EpicsHostArch.pl
@@ -26,7 +26,10 @@ sub GetEpicsHostArch { # no args
     if ($arch =~ /sun4-solaris/)        { return "solaris-sparc";
     } elsif ($arch =~ m/i86pc-solaris/) { return "solaris-x86";
     } elsif ($arch =~ m/i[3-6]86-linux/){ return "linux-x86";
-    } elsif ($arch =~ m/x86_64-linux/)  { return "linux-x86_64";
+     elsif ($arch =~ m/x86_64-linux/)  {
+        my($kernel, $hostname, $release, $version, $cpu) = POSIX::uname();
+        if ( $release=~ m/cct$/)        { return "linux-corei7-poky"; }
+        else                            { return "linux-x86_64"; }
     } elsif ($arch =~ m/arm-linux/)     { return "linux-arm";
     } elsif ($arch =~ m/MSWin32-x86/)   { return "win32-x86";
     } elsif ($arch =~ m/MSWin32-x64/)   { return "windows-x64";
@@ -40,7 +43,12 @@ sub GetEpicsHostArch { # no args
             elsif ($cpu =~ m/i386/)         { return "darwin-x86";  }
             elsif ($cpu =~ m/x86_64/)       { return "darwin-x86";  }
             else                            { return "unsupported"; }
-    } else { return "unsupported"; }
+    } elsif ($arch =~m/powerpc64-linux/)    {
+	my($kernel, $hostname, $release, $version, $cpu) = POSIX::uname();
+	if ($cpu =~ m/ppc64/)               { return "linux-ppc64e6500";  }
+	else                                { return "unsupported"; }
+    }
+    else { return "unsupported"; }
 }
 
 #EOF EpicsHostArch.pl
