diff --git startup/EpicsHostArch.pl startup/EpicsHostArch.pl
index 09f7ffd..cb9dfec 100755
--- startup/EpicsHostArch.pl
+++ startup/EpicsHostArch.pl
@@ -40,7 +40,12 @@ sub GetEpicsHostArch { # no args
             elsif ($cpu =~ m/i386/)         { return "darwin-x86";  }
             elsif ($cpu =~ m/x86_64/)       { return "darwin-x86";  }
             else                            { return "unsupported"; }
-    } else { return "unsupported"; }
+    } elsif ($arch =~m/powerpc64-linux/)    {
+	my($kernel, $hostname, $release, $version, $cpu) = POSIX::uname();
+	if ($cpu =~ m/ppc64/)               { return "linux-ppc64e6500";  }
+	else                                { return "unsupported"; }
+    }
+    else { return "unsupported"; }
 }
 
 #EOF EpicsHostArch.pl
