diff --git configure/RULES_BUILD configure/RULES_BUILD
index 2949ed9c2..71dff4058 100644
--- configure/RULES_BUILD
+++ configure/RULES_BUILD
@@ -307,7 +307,7 @@ $(LOADABLE_SHRLIBNAME): $(LOADABLE_SHRLIB_PREFIX)%$(LOADABLE_SHRLIB_SUFFIX):
 
 $(LIBNAME) $(SHRLIBNAME) $(LOADABLE_SHRLIBNAME): | $(INSTALL_LIB)
 $(INSTALL_LIB):
-	@$(MKDIR) $@
+#	@$(MKDIR) $@
 
 #---------------------------------------------------------------
 # C++ munching for VxWorks
